lib/ansible/modules/network/f5/bigip_static_route.py

```diff
@@ -286,22 +295,23 @@ def destination_ip(self):
     def netmask(self):
         if self._values['netmask'] is None:
             return None
-        # Check if numeric
+        result = -1
         try:
             result = int(self._values['netmask'])
-            if 0 <= result < 256:
-                return result
+            if 0 < result < 256:
+                pass
+        except ValueError:
+            if is_valid_ip(self._values['netmask']):
```

Comment:
Why are you replacing a valid [EAFP-style](https://stackoverflow.com/a/11360880/595220) style check with if-clause?